api/controllers/v1/massif/get-dashboard-data.js

```diff
@@ -0,0 +1,60 @@
+// const ControllerService = require('../../../services/ControllerService');
+const ErrorService = require('../../../services/ErrorService');
+const DashboardDataMassifService = require('../../../services/DashboardDataMassifService');
+
+module.exports = async (req, res) => {
+  try {
+    // TODO : check if id is present
+    await DashboardDataMassifService.createMaterializedView();
+
+    const caveWithMaxDepth =
+      DashboardDataMassifService.getCaveWithMaxDepthInMassif(req.params.id);
+
+    const nbCaves = DashboardDataMassifService.getNbCavesInMassif(
+      req.params.id
+    );
+
+    const caveWithMaxLength =
+      DashboardDataMassifService.getCaveWithMaxLengthInMassif(req.params.id);
+
+    const nbCavesWhichAreDiving =
+      DashboardDataMassifService.getNbCavesWhichAreDivingInMassif(
+        req.params.id
+      );
+
+    const avgDepthAndLength =
+      DashboardDataMassifService.getAvgDepthAndLengthInMassif(req.params.id);
+
+    const totalLength = DashboardDataMassifService.getTotalLength(
+      req.params.id
+    );
+
+    return Promise.all([
+      nbCaves,
+      caveWithMaxDepth,
+      caveWithMaxLength,
+      nbCavesWhichAreDiving,
+      avgDepthAndLength,
+      totalLength,
+    ]).then((values) => {
```

Comment:
You can use `await` instead of `Promise.all()` to avoid the `then` callback: it's clearer.